x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_field_maps.ts

```diff
@@ -76,3 +76,9 @@ export const prebuiltRulesFieldMap: FieldMap<SchemaFieldMapKeys<RuleMigrationPre
   rule_id: { type: 'keyword', required: true },
   mitre_attack_ids: { type: 'keyword', array: true, required: false },
 };
+
+export const migrationsFieldMaps: FieldMap<SchemaFieldMapKeys<SiemMigration>> = {
+  id: { type: 'keyword', required: true },
```

Comment:
Do we need an `id` mapped field?

Reply:
🤦 .. you are right. Fixed in 60e84507f31